charmcraft/commands/analyze.py

```diff
@@ -54,6 +57,11 @@ def fill_parser(self, parser):
             action="store_true",
             help="Force to run all checks, even those set to ignore in the configuration",
         )
+        parser.add_argument(
+            "--format",
+            choices=[JSON_FORMAT],
```

Comment:
Will we offer other formats in the future?

Reply:
Maybe! No plans by now, though.